src/main/liquibase/liquibase-ciel-data.xml

```diff
@@ -49,8 +47,8 @@
 	</changeSet>
 
 	<changeSet id="20120315-1100" author="standalone" dbms="mysql">
-		<comment>Adding mvp data</comment>
```

Comment:
Before this line please add the line `<validChecksum>3:da27117b32a07c5b89b9df072d8d3b45</validChecksum>`